homeworks/franktoth/4_exceptions_logging/to_do.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import os
+
+# Create the tasks.txt file if it doesn't exist
+if not os.path.exists("tasks.txt"):
+    with open("tasks.txt", "w") as f:
+        pass
+
+# Configure logging to write to both console and a log file
```

Comment:
Így egyelőre csak a file-ba logol, ne felejtsd el a konzol-ra való logolást sem.
Ezen felül kérlek abba a mappába kerüljön a log file ahol a to_do.py is van, ne a root-ba. 
Simán cseréld le a filename='todo.log' részt egy relative path-re a megfelelő elérési útvonalhoz.

Reply:
Mit jelent a konzolra való logolás? írja ki minden log entry esetén a képernyőre azt a sort, amit a file-ban rögzít?
A log file-nál megvan a relative path.

Reply:
Szia! Ne haragudj ezt a kommentedet csak most láttam, hogy újra átfutottam a PR-odat. 
Nem feltétlenül jelenti azt, hogy ugyanazt a sort írja ki amit a file-ba is logol. Vannak logging szintek amiket be tudsz állítani külön a filehandler-nél és külön a streamhandler-nél. Lehetséges hogy a log file-ba minden szintű log bekerüljön (DEBUG-tól felfelé minden), míg a a konzolra (streamhandler) pedig csak INFO és fölötte. Ez egyéni preferencia, a lényeg igazából itt csak az, hogy használd mindkettőt valamilyen beállítással.

Reply:
Szia!
Köszi a magyarázatot! Beírtam ahogy szerintem jó.
Üdv:
Frank